draft-irtf-panrg-path-properties.md

```diff
@@ -166,17 +170,15 @@ Another example is a connection which is composed of multiple streams where each
 
 This Section gives some examples of path properties which may be useful, e.g., for the use cases described in {{use-cases}}.
 
+Within the context of any particular technology, entities may or may not have insight into and be able to influence specific path elements and their path properties.
+Therefore, the path properties available in each such context may differ.
+For example, a host may have some visibility into path elements that are close, i.e., within the first few hops, or they may include path elements that are far away and/or on a different level of abstraction, e.g., the list of ASes traversed.
+The visibility of path elements and path properties to a specific entity may depend on factors such as the physical or network distance or the existence of trust or contractual relationships between the entity and the path element(s).
```

Comment:
Maybe we can rephrase this paragraph more concisely:
"Within the context of a particular technology, available path properties may differ as entities have insight into and are able to influence different path elements and path properties.
For example, a host may have some visibility into path elements that are close, i.e., within the first few hops, or they may include path elements that are far away and/or on a different level of abstraction, e.g., the list of ASes traversed.
Additionally, the visibility may depend on factors such as the physical or network distance or the existence of trust or contractual relationships between the entity and the path element(s)."
_should we also use **host** in the last sentence as in the sentence before?_

Reply:
Thanks, I used the first sentence you suggested.
Then, re-reading the second sentence, I rephrased it a bit, as I think its grammar was broken and I wanted more symmetry between the two options (low VS higher abstraction level and being close VS far away). Please let me know what you think.
For the last sentence, yes, let's stay with the example and use "host" here. But I intended this sentence to be a possible explanation for the sentence before, so it's not really "Additionally".

Reply:
The text looks good now